docs/xml/elements/hours_open.rst

```diff
@@ -1,57 +1,103 @@
 HoursOpen
 =========
 
-.. todo::
+A structured way of describing the days and hours that a place such as a
+:doc:`Office <office>` or :doc:`PollingLocation <polling_location>` is open, or
+that an event such as an :doc:`Election <election>` is happening.
 
-   Document HoursOpen & Schedule
++----------+--------------------+------------+----------+-----------------------+----------------------------------+
+| Tag      | Data Type          | Required?  | Repeats? |Description            |Error Handling                    |
+|          |                    |            |          |                       |                                  |
++==========+====================+============+==========+=======================+==================================+
+| Schedule | Schedule           |**Required**| Repeats  |Defines a block of days|At least one valid `Schedule`_    |
```

Comment:
Sorry didn't notice this before. Just need a link for "Schedule" in the Data Type column, as well.